java/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.4)
```

Comment:
Unfortunately, Travis seems only to have version 3.2.2, so I might have to rethink this approach (or figure out how to get 3.4 on Travis).